### PR TITLE
Add lookahead in all-in-one

### PR DESCRIPTION
**Pull request by hxsz1997**
Add lookahead in all-in-one.

### CODE REVIEW

File: python/llm/dev/benchmark/all-in-one/run.py [hunk 1]
```diff
@@ -460,34 +472,28 @@ def run_transformer_int4_gpu(repo_id,
...
             input_list = [true_str] * batch_size
             input_ids = tokenizer(input_list, return_tensors="pt").input_ids.to('xpu')
             actual_in_len = input_ids.shape[1]
             result[in_out] = []
-            thread = threading.Thread(target=run_model_in_thread, args=(model, in_out, tokenizer, result, warm_up, num_beams, input_ids, out_len, actual_in_len, num_trials, load_time))
+            thread = threading.Thread(target=run_model_in_thread, args=(model, in_out, tokenizer, result, warm_up, num_beams, input_ids, out_len, actual_in_len, num_trials, load_time, lookup))
             thread.start()
             thread.join()
 
-            if result[in_out]:
```
Comment (cyita) [inline python/llm/dev/benchmark/all-in-one/run.py:476]: Please don't remove this code snippet.

File: python/llm/dev/benchmark/all-in-one/run.py [hunk 2]
```diff
@@ -109,6 +117,8 @@ def run_model(repo_id, test_api, in_out_pairs, local_model_hub=None, warm_up=1,
         result = run_speculative_gpu(repo_id, local_model_hub, in_out_pairs, warm_up, num_trials, num_beams, batch_size)
     elif test_api == 'pipeline_parallel_gpu':
         result = run_pipeline_parallel_gpu(repo_id, local_model_hub, in_out_pairs, warm_up, num_trials, num_beams, low_bit, batch_size, cpu_embedding, fp16=use_fp16_torch_dtype, n_gpu=n_gpu)
+    elif test_api == "transformer_int4_fp16_lookahead_gpu":
+        result = run_transformer_int4_gpu(repo_id, local_model_hub, in_out_pairs, warm_up, num_trials, num_beams, low_bit, batch_size, cpu_embedding, fp16=True, lookup=True)
```
Comment (cyita) [inline python/llm/dev/benchmark/all-in-one/run.py:121]: lookup -> lookahead.

Comment (hxsz1997) [reply python/llm/dev/benchmark/all-in-one/run.py]: Sure. I have changed lookup to lookahead.

File: python/llm/dev/benchmark/all-in-one/run.py (hunk 1 above)
Comment (hxsz1997) [reply python/llm/dev/benchmark/all-in-one/run.py]: Sure. I have fixed it.

File: python/llm/dev/benchmark/all-in-one/config.yaml [hunk 3]
```diff
@@ -12,7 +12,7 @@ in_out_pairs:
   - '32-32'
   - '1024-128'
 test_api:
-  - "transformer_int4_gpu"                # on Intel GPU, transformer-like API, (qtype=int4)
+  # - "transformer_int4_gpu"                # on Intel GPU, transformer-like API, (qtype=int4)
```
Comment (cyita) [inline python/llm/dev/benchmark/all-in-one/config.yaml:15]: Set "transformer_int4_gpu" as the default option.

**Pull request merged**